code/game/objects/items/weapons/clown.dm

```diff
@@ -0,0 +1,44 @@
+
+/obj/item/weapon/pie_cannon
```

Comment:
this is kind of disgusting tbh.

It should really just be a special pneumatic cannon that only takes pies (then you don't need this projectile nonsense)